fetch.js

```diff
@@ -186,7 +186,9 @@
     }
 
     this.json = function() {
-      return this.text().then(JSON.parse)
+      return this.text().then(function(text) {
+        return JSON.parse(text)
+      })
```

Comment:
I'm :+1: for merging this change because it's innocent (adds an overhead of calling 1 more anon function) and fixes the current Firefox problem that might continue biting users that haven't upgraded to a future Firefox that may fix this.